libs/common/src/abstractions/organization-user/organization-user.service.ts

```diff
@@ -94,6 +95,18 @@ export abstract class OrganizationUserService {
     ids: string[]
   ): Promise<ListResponse<OrganizationUserBulkResponse>>;
 
+  /**
+   * Accept an organization user invitation and initialize the organization
```

Comment:
I think this needs a bit more explanation to make it clear this isn't the standard flow.

```suggestion
   * Accept an invitation to initialize and join an organization created via the Admin Portal **only**.
   * This only used once for the initial Owner, because it includes the organization's encryption keys.
   * This should not be used for organizations created via the Web client.
```